include/hw/wb_fofb_processing_regs.h

```diff
@@ -23,17 +23,23 @@
 #define WBGEN2_SIGN_EXTEND(value, bits) (((value) & (1<<bits) ? ~((1<<(bits))-1): 0 ) | (value))
 #endif
 
-#define FOFB_PROCESSING_REGS_RAM_BANK_SIZE                                     \
-  (WB_FOFB_PROCESSING_REGS_RAM_BANK_1 - WB_FOFB_PROCESSING_REGS_RAM_BANK_0)
-
-#define FOFB_PROCESSING_REGS_RAM_BANK_OFFS(chan)                               \
-  (WB_FOFB_PROCESSING_REGS_RAM_BANK_0 + FOFB_PROCESSING_REGS_RAM_BANK_SIZE*chan)
-
-#define FOFB_PROCESSING_REGS_REG_FIXED_POINT_POS    WB_FOFB_PROCESSING_REGS_FIXED_POINT_POS
-#define FOFB_PROCESSING_REGS_FIXED_POINT_POS        FOFB_PROCESSING_REGS_REG_FIXED_POINT_POS
-#define FOFB_PROCESSING_REGS_FIXED_POINT_POS_MASK   WB_FOFB_PROCESSING_REGS_FIXED_POINT_POS_VAL_MASK
-#define FOFB_PROCESSING_REGS_FIXED_POINT_POS_SHIFT  WB_FOFB_PROCESSING_REGS_FIXED_POINT_POS_VAL_SHIFT
-#define FOFB_PROCESSING_REGS_FIXED_POINT_POS_R(val) ((val & FOFB_PROCESSING_REGS_FIXED_POINT_POS_MASK) >> FOFB_PROCESSING_REGS_FIXED_POINT_POS_SHIFT)
-#define FOFB_PROCESSING_REGS_FIXED_POINT_POS_W(val) ((val & FOFB_PROCESSING_REGS_FIXED_POINT_POS_MASK) << FOFB_PROCESSING_REGS_FIXED_POINT_POS_SHIFT)
+#define FOFB_PROCESSING_REGS_COEFFS_RAM_BANK_SIZE                                     \
+  (WB_FOFB_PROCESSING_REGS_COEFFS_RAM_BANK_1 - WB_FOFB_PROCESSING_REGS_COEFFS_RAM_BANK_0)
+
+#define FOFB_PROCESSING_REGS_COEFFS_RAM_BANK_OFFS(chan)                               \
+  (WB_FOFB_PROCESSING_REGS_COEFFS_RAM_BANK_0 + FOFB_PROCESSING_REGS_COEFFS_RAM_BANK_SIZE*chan)
+
+#define FOFB_PROCESSING_REGS_SETPOINTS_RAM_BANK_SIZE                           \
+  (WB_FOFB_PROCESSING_REGS_SETPOINTS_RAM_BANK - WB_FOFB_PROCESSING_REGS_COEFFS_RAM_BANK_11)
+
+#define FOFB_PROCESSING_REGS_SETPOINTS_RAM_BANK_OFFS                           \
+  (WB_FOFB_PROCESSING_REGS_SETPOINTS_RAM_BANK)
+
+#define FOFB_PROCESSING_REGS_REG_COEFFS_FIXED_POINT_POS     WB_FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS
+#define FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS         FOFB_PROCESSING_REGS_REG_COEFFS_FIXED_POINT_POS
+#define FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_MASK    WB_FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_VAL_MASK
+#define FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_SHIFT   WB_FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_VAL_SHIFT
+#define FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_R(val)  ((val & FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_MASK) >> FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_SHIFT)
+#define FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_W(val)  ((val & FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_MASK) << FOFB_PROCESSING_REGS_COEFFS_FIXED_POINT_POS_SHIFT)
```

Comment:
We shouldn't need to use this macro, since this register is read only, but the definition is wrong; it should be `(val << shift) & mask`.